### PR TITLE
Fixed the vanilla client attempting to chunk blend a Dragonfly world

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

File: server/world/mcdb/provider.go [hunk 1]
```diff
@@ -28,6 +28,13 @@ type Provider struct {
 // chunkVersion is the current version of chunks.
 const chunkVersion = 27
 
+var (
+	// lastOpenedWithVersion is the current last opened with version, required by the latest Minecraft data provider.
+	lastOpenedWithVersion = []int32{1, 18, 0, 2, 0}
+	// minimumCompatibleClientVersion is the minimum compatible client version, required by the latest Minecraft data provider.
+	minimumCompatibleClientVersion = []int32{1, 18, 0, 0, 0}
```
Comment (Sandertv) [inline server/world/mcdb/provider.go:35]: Do you reckon we could base these on the protocol version? I'm not a huge fan of having to update these everytime. I suppose the minimum might not be easy to do but with the lastOpenedWithVersion that should be possible(?).

Comment (Sandertv) [reply server/world/mcdb/provider.go:35]: Also are all 3 of NetworkVersion, LastOpenedWithVersion and MinimumCompatibleClientVersion needed for it to work?

Comment (JustTalDevelops) [reply server/world/mcdb/provider.go:35]: I haven't tried each one of them separately, but honestly better to be safe then sorry. I can look into basing these off of protocol.CurrentVersion though.

Comment (Sandertv) [reply server/world/mcdb/provider.go:35]: That would be a good improvement.

Comment (Sandertv) [reply server/world/mcdb/provider.go:35]: Any update on this? Do you reckon we could resolve this and get this merged by the end of the month?

Comment (JustTalDevelops) [reply server/world/mcdb/provider.go]: Updated.

**Comment by JustTalDevelops** [comment]
Another review on this would be great.

**Review by Sandertv — APPROVED** [review]
Looks good to me!

**Pull request merged**